.github/CONTRIBUTING.md

```diff
@@ -95,6 +96,24 @@ Refactoring code shouldn't require any new tests, but you should [make sure the
 
 [Done with your refactoring?](#i-finished-my-changes)
 
+## How to update dependencies
+
+[Greenkeeper](https://greenkeeper.io) keeps track of dependencies for us and submits pull requests when a dependency is updated. To update a depencency, you will need to follow these steps:
+
+1. The Pull Request will contain useful information about what's changed, including breaking changes. You will also want to check the dependency's github and/or npm listing for changelog and recommended updating steps.
+
+2. Pull down the branch locally.
+
+3. Update the dependency and perform any steps the project suggests for updating. Sometimes this requires updating other dependencies, changing configuration code, or other tasks.
+
+4. Build, run, and verify that tests pass.
+
+5. Verify that everything looks good- Check pages and components that make use of the library and ensure they are still working as expected.
+
+6. Rebase, Squash, and push your changes to github.
+
+At this point, follow the steps described in [I finished my changes](#i-finished-my-changes), but you won't need to open a Pull Request since one already exists. Mark the Pull Request as `Needs Code Review`. 
```

Comment:
Prefer lowercased `pull request` and `needs code review`. This includes similar instances above.